### PR TITLE
add vanilla swap

### PR DESCRIPTION
**Pull request by rogi-sh**
##### Name (to be shown on DefiLlama):
VanillaSwap

##### Twitter Link:
@VanillaSwap1

##### Website Link:
https://vanillalabs.org/
https://v2.vanillaswap.org/#/swap

##### Logo (High resolution, will be shown with rounded borders):
image

##### Current TVL:
629k

##### Chain:
1130 defichain_evm

##### Short Description (to be shown on DefiLlama):
VanillaSwap is a DEX that allows users to be Liquidity Provider, Farming, Staking, Exchange Assets on the DeFiMetachain and more...

##### Category (full list at https://defillama.com/categories) *Please choose only one:
DEXES

##### forkedFrom (Does your project originate from another project):
Uni Swap

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/vanillaswap-v2 exports TVL: 
        
 
 
    defichain_evm             0
    
    total                    0

**Comment by llamatester** [comment]
The adapter at projects/vanillaswap-v2 exports TVL: 
        
 
 
    defichain_evm             0
    
    total                    0

**Comment by llamatester** [comment]
The adapter at projects/vanillaswap-v3 exports TVL: 
        
 
 
    defichain_evm             0
    
    total                    0

**Comment by llamatester** [comment]
The adapter at projects/vanillaswap-v2 exports TVL: 
        
 
 
    defichain_evm             0
    
    total                    0

**Comment by llamatester** [comment]
The adapter at projects/vanillaswap-v3 exports TVL: 
        
 
 
    core                      0
    
    total                    0

**Comment by rogi-sh** [comment]
How i can set the correct chain: defichain_evm in v3?

File: projects/vanillaswap-v3/index.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+const { uniV3Export } = require("../helper/uniswapV3");
+const factory = "0x79Ea1b897deeF37e3e42cDB66ca35DaA799E93a3";
+
+module.exports = uniV3Export({
+    core: { factory, fromBlock: 512861 },
```
Comment (Define101) [inline projects/vanillaswap-v3/index.js:5]: replace core for defichain_evm

**Comment by llamatester** [comment]
The adapter at projects/vanillaswap-v2 exports TVL: 
        
 
 
    defichain_evm             0
    
    total                    0

**Comment by llamatester** [comment]
The adapter at projects/vanillaswap-v3 exports TVL: 
        
 
 
    defichain_evm             0
    
    total                    0

**Comment by llamatester** [comment]
The adapter at projects/vanillaswap-v2 exports TVL: 
        
 
 
    defichain_evm             0
    
    total                    0

**Comment by llamatester** [comment]
The adapter at projects/vanillaswap-v3 exports TVL: 
        
 
 
    defichain_evm             0
    
    total                    0

File: projects/vanillaswap-v3/index.js (hunk 1 above)
Comment (rogi-sh) [reply projects/vanillaswap-v3/index.js]: done

**Comment by llamatester** [comment]
The adapter at projects/vanillaswap-v2 exports TVL: 
        
 
 
    defichain_evm             438.45 k
    
    total                    438.45 k

**Comment by llamatester** [comment]
The adapter at projects/vanillaswap-v3 exports TVL: 
        
 
 
    defichain_evm             37.61 k
    
    total                    37.61 k

**Comment by llamatester** [comment]
The adapter at projects/vanillaswap-v2 exports TVL: 
        
 
 
    defichain_evm             438.45 k
    
    total                    438.45 k

**Comment by llamatester** [comment]
The adapter at projects/vanillaswap-v3 exports TVL: 
        
 
 
    defichain_evm             272.38 k
    
    total                    272.38 k

**Pull request merged**